package.json

```diff
@@ -67,7 +67,7 @@
     "cp-file": "^4.2.0",
     "cpy": "^5.1.0",
     "cssstats": "^3.1.0",
-    "eslint": "^4.9.0",
+    "eslint": "^4.17.0",
```

Comment:
FYI this is probably unnecessary - this was one of my attempts to get the ESLint tests to work